pkg/odo/cli/service/create.go

```diff
@@ -1,62 +1,43 @@
 package service
 
 import (
-	"bytes"
 	"errors"
 	"fmt"
-	"strings"
-	"text/template"
-
 	"github.com/openshift/odo/pkg/log"
 	"github.com/openshift/odo/pkg/odo/cli/component"
-	"github.com/openshift/odo/pkg/odo/cli/service/ui"
 	"github.com/openshift/odo/pkg/odo/genericclioptions"
-	"github.com/openshift/odo/pkg/odo/util/completion"
 	"github.com/spf13/cobra"
 	ktemplates "k8s.io/kubectl/pkg/util/templates"
+	"strings"
 )
 
 const (
 	createRecommendedCommandName = "create"
-	equivalentTemplate           = "{{.CmdFullName}} {{.ServiceType}}" +
-		"{{if .ServiceName}} {{.ServiceName}}{{end}}" +
-		" --app {{.Application}}" +
-		" --project {{.Project}}" +
-		"{{if .Plan}} --plan {{.Plan}}{{end}}" +
-		"{{range $key, $value := .ParametersMap}} -p {{$key}}={{$value}}{{end}}"
 )
 
 var (
-	createExample = ktemplates.Examples(`
-    # Create new postgresql service from service catalog using dev plan and name my-postgresql-db.
-    %[1]s dh-postgresql-apb my-postgresql-db --plan dev -p postgresql_user=luke -p postgresql_password=secret`)
-
 	createOperatorExample = ktemplates.Examples(`
 	# Create new EtcdCluster service from etcdoperator.v0.9.4 operator.
 	%[1]s etcdoperator.v0.9.4/EtcdCluster`)
 
 	createShortDesc = `Create a new service from Operator Hub or Service Catalog and deploy it on OpenShift.`
```

Comment:
```suggestion
    createShortDesc = `Create a new service from Operator Hub and deploy it on OpenShift.`
```

Reply:
On an unrelated note: Why do we just mention OpenShift, and not K8s in our help docs?

Reply:
yea i thought so too. Makes sense to me. We should probably have an issue to update all cli docs to mention kubernetes. Openshift is a kind of kubernetes after all

Reply:
No i think we still need to check if csv is supported. Operator hub is not installed by default on a cluster

Reply:
I believe some of the references to the service catalog can be interpreted as the `odo catalog list services` = a service catalog.

Reply:
@valaparthvi ^ I will go through your links once more and see which of the references makes sense and which does not

Reply:
@mohammedzee1000 You missed out on this change.